### ISSUE TITLE
[Accessibility] Keyboard navigation does not work correctly on the Heatmap in the 9.3 version.

### ISSUE THREAD

**Issue by magdalena-gut** [highcharts/highcharts#16549]
#### Expected behaviour
Tab key should navigate between the chart elements. Arrow keys should navigate between the points.

#### Actual behaviour
Tab key behaviour is correct, but the arrow keys do not navigate between points on the Heatmap.
The latest working version is 9.2.2.

#### Live demo with steps to reproduce
Use the tab key to choose a chart to navigate and try to navigate between the points using arrow keys.

Demo:
https://jsfiddle.net/BlackLabel/kh50Lzc2/

#### Product version
Highcharts 9.3.0

#### Affected browser(s)
N/A

**Comment by bre1470** [comment]
I am not able to reproduce a working version with v9.2.1 or v9.2.2.

**Comment by bre1470** [comment]
To clarify: Screen reader with arrow keys is working in all versions. Tab and arrow keys without active screen reader do not work (differently) between/in v9.2.1, v9.2.2 and v9.3.0.

**Comment by raf18seb** [comment]
@bre1470 maybe it's Windows 10 related (tested on both Chrome and Firefox)

In 9.2.2 it is enough to press `tab` - the legend is focused, then `tab+shift` - the point is focused, and from this place it is possible to navigate only by using arrow keys.
See the gif
![heatmap-accessibility-bug](https://user-images.githubusercontent.com/23310742/140064684-60d95972-e6b1-49a1-b026-9dd7a23900aa.gif)

But in v9.3.0 it is very hard to make navigation work when using only arrows. I was able to make it work after many `tab`, `shift`, `arrows` combinations, but I couldn't find a pattern.
See below. On this gif, I was trying `tab` and `tab+shift` combinations with pressing arrows all the time (like 100 times during this gif), but I couldn't navigate properly.

![heatmap-accessibility-bug-2](https://user-images.githubusercontent.com/23310742/140065107-b6e9d0c4-8d5e-4f1b-9b83-e322eea01720.gif)
:

**Comment by bre1470** [comment]
@raf18seb Thank you for the detailed explanation. I will try tomorrow with Windows in BS.

**Comment by TorsteinHonsi** [comment]
@raf18seb Have you managed to bisect it to a specific commit?

**Comment by bre1470** [comment]
Three things I figured out so far:
- In Windows Edge one can surely navigate with `tab` followed by an `up/down/left/right`, and again `tab` + arrow key etc. So this indicates a focus problem as pressing tab ones fixes it.
- Migration of the KeyboardNavigation did not create the issue.
- Order of composition calls seems to have no effect.

**Comment by bre1470** [comment]
Bisect to https://github.com/highcharts/highcharts/commit/cb068fa3a3877a0be6369668e4b1536843c2396f
